### PR TITLE
435 suggest url based on heading

### LINKED ISSUE #435: Suggest Page URL based on Heading
Right now the URL is created as a long string, for example "http://friedhelm.uwazi.io/page/4db5883ef6e4fc7cdbcdf8f7150888d8" and it may be preferable to suggest a link based on the heading, e.g. in this case "http://friedhelm.uwazi.io/page/about-this-collection". Makes it easier for users and should help with search engine optimisation.

### PR DESCRIPTION
**Pull request by vkozinec**
fixes #435 

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

**Review by fnocetti — CHANGES_REQUESTED** [review]
Hi @vkozinec ! Thank you for the PR and please excuse the delay in the review.
The code looks good and the implementation runs as expected.
I left one comment that I'd appreciate you to take a look. Also there's an e2e failing in the puppeteer suite. Could you please take a look?

Please let us know if you need any assistance. Thanks!

File: app/react/Pages/components/PageCreator.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { Form, Field, Control } from 'react-redux-form';
 import { bindActionCreators, Dispatch } from 'redux';
 import { connect, ConnectedProps } from 'react-redux';
 import React, { Component } from 'react';
+import _ from 'lodash';
```
[the remark this answers is not in the document]
Comment (ivan-init) [reply app/react/Pages/components/PageCreator.tsx:5]: @fnocetti Lodash is now installed as a dependency and e2e tests are fixed

[the remark this answers is not in the document]
Comment (yacky) [reply app/react/Pages/components/PageCreator.tsx:5]: @fnocetti Lodash is now installed as a dependency and e2e tests are fixed

[the remark this answers is not in the document]
Comment (fnocetti) [reply app/react/Pages/components/PageCreator.tsx:5]: Thanks @yacky, I'll take a look!

**Review by fnocetti — APPROVED** [review]
LGTM

**Pull request merged**